cmake/Macros.cmake

```diff
@@ -50,10 +50,10 @@ function(sfml_set_common_ios_properties target)
 endfunction()
 
 # add a new target which is a SFML library
-# example: sfml_add_library(sfml-graphics
+# example: sfml_add_library(graphics
```

Comment:
I wonder, if we're using capitalized module names everywhere now, shouldn't this also work as such and reverse the functionality internally to lowercase the module name?

Reply:
Do you mean change the convention to pass in an upper case module name to `sfml_add_library` like `sfml_add_library(Graphics)` then convert that to lower case before creating the internally used `sfml-graphics` style CMake target names? That would certainly make for a simpler implementation of `sfml_add_library` since converting strings to lowercase is a lot more simple than uppercasing the first letter.

Reply:
Yep, that's what I meant

Reply:
I like that idea. I'll try implementing it to make sure it's as simple as I'm expecting it to be.

Reply:
All done!